src/components/OSCALControlProse.js

```diff
@@ -412,14 +414,14 @@ export function OSCALReplacedProseWithByComponentParameterValue(props) {
       }
       if (isEditingStatement) {
         // We're currently editing this statement, so build param input
-        // TODO - Populate implementationSetParameters with values from statementByComponent or empties
+        // TODO - Populate implementationSetParameters with values from statementByComponent or empties (https://github.com/EasyDynamics/oscal-react-library/issues/503)
         const setParameter = statementByComponent?.["set-parameters"]?.find(
           (element) => element["param-id"] === segment
         ) ?? { "param-id": segment };
         setParameter.values ??= [null];
         const setParameterRef = setParametersRefs[segment];
 
-        // TODO - support for more than 1 item in values arrays
+        // TODO - support for more than 1 item in values arrays (https://github.com/EasyDynamics/oscal-react-library/issues/501)
```

Comment:
For example here the code itself explains that we allow only 1 item and the issue referencing the code can then be used to tell us we need to support multiple values.

Reply:
That makes sense! After this next commit, all of the `TODO`s will have GitHub issues that are linked to the specific part in the code base so I'll remove the references to the `TODO`s.